.github/workflows/builds-canary.yaml

```diff
@@ -0,0 +1,36 @@
+name: Canary builds
+
+on:
+  workflow_run:
+    workflows:
+      - CI tests
+    branches:
+      - master
+    types:
+      - completed
+
+jobs:
+  build:
+    if: github.event.workflow_run.conclusion == 'success'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            subdir: linux-64
+          - runner: macos-latest
+            subdir: osx-64
+          - runner: windows-latest
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Should we include these in the composite action as well?

Reply:
Ah, yeah, good call, this makes sense, since we're expecting a conda environment might as well as creating it there. 👍🏻